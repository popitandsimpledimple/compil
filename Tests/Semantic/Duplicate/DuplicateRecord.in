type point = record x: integer; 
                    x: integer; end;
begin end.