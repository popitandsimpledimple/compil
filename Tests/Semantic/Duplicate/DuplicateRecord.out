(2,21) Duplicate identifier 'x'